temporal-sdk/src/main/java/io/temporal/internal/statemachines/WorkflowTaskStateMachine.java

```diff
@@ -108,30 +109,29 @@ private WorkflowTaskStateMachine(long workflowTaskStartedEventId, Listener liste
                   WorkflowTaskStateMachine::handleFailed)
               .add(State.STARTED, EventType.EVENT_TYPE_WORKFLOW_TASK_TIMED_OUT, State.TIMED_OUT);
 
+  private synchronized void startWorkflowTaskOnce(boolean lastTaskInHistory) {
+    if (!workflowTaskStarted) {
+      listener.workflowTaskStarted(
+          startedEventId, eventTimeOfTheLastWorkflowStartTask, lastTaskInHistory);
+      workflowTaskStarted = true;
```

Comment:
Should we call this `listenerNotifiedWorkflowTaskStarted` to make it more explicit?